exercises/calculator.cpp

```diff
@@ -10,3 +10,36 @@
   Multiplication: 8
   Division: 2
 */
+
+#include <iostream>
+
+using namespace std;
+
+template<typename T>sum(T a, T b){
+	return a+b;
+}
+
+template<typename T>difference(T a, T b){
+	return a-b;
+}
+
+template<typename T>multiplication(T a, T b){
+	return a*b;
+}
```

Comment:
not bad 👍 

io metterei uno spazio tra `){` oppure tra `a*b` o `a+b` ecc. ma questione di stile
personalmente penso che gli spazi `aumentinolaleggibilità`, scrivere tutto unito non aiuta l'occhio a leggere il codice